test/functionalTest/cases/1259_new_json_for_v2/put_attribute_value.test

```diff
@@ -30,6 +30,9 @@ brokerStart CB 0-255
 --SHELL--
 
 #
+# FIXME P6: steps 04, 06, 08, 10, 12, 14, 16 18, and 20 in REGEXPECT of this test must change once the
```

Comment:
Probably we should close #2451 soon (I think it is quite clear no modification is needed, see my comment on that issue: https://github.com/telefonicaid/fiware-orion/issues/2451#issuecomment-242446354) thus removing these 3 lines before merging this PR.